collect_app/src/main/res/drawable/form_state_saved_circle.xml

```diff
@@ -4,14 +4,11 @@
         android:viewportWidth="120"
         android:viewportHeight="120">
 
-    <path
```

Comment:
Removing the colors from the form status icons could definitely be controversial. I've got a few thoughts on this:

* We view forms by their status ("Fill Blank Form", "Edit Saved Form" etc) so the icons are mostly going to be the same. The only exceptions are failed submissions (which I've kept the colouring on) and Edit Saved Form containing finalized and non-finalized forms (which we're planning to get rid of).
* The color is only fuzzily linked to the state and will have different implications for different people.
* In my opinion, the color actually distracts from the icon itself, which I think conveys a little more about the state.
* Generally, I think the app looks cleaner without the coloured icons.